### PR TITLE
PAYARA-1378 create hazelcast instance command

### PR DESCRIPTION
**Pull request by Cousjava**

### CODE REVIEW

**Comment by Cousjava** [comment]
jenkins test please

File: nucleus/payara-modules/hazelcast-bootstrap/pom.xml [hunk 1]
```diff
@@ -78,5 +78,11 @@
             <version>${lombok.version}</version>
             <scope>provided</scope>
         </dependency>
+        <dependency>
+            <groupId>org.glassfish.main.admin</groupId>
+            <artifactId>admin-cli</artifactId>
+            <version>4.1.1.172-SNAPSHOT</version>
```
Comment (smillidge) [inline nucleus/payara-modules/hazelcast-bootstrap/pom.xml]: Dependency here should be ${project.version}

**Comment by Cousjava** [comment]
jenkins test please

**Comment by Cousjava** [comment]
jenkins test please

**Pull request merged**